01 - Community/People/jdanielmourao.md

```diff
@@ -17,15 +17,15 @@ publish: true
 
 
 ## Author of
-
 %% Begin Hub: Released contributions %%
 
 ### Themes
 - [[Sanctum]]
 %% End Hub: Released contributions %%
 
 %% Add links to any plugins, themes or other notes that the author has created but are not (yet) included in the `obsidian-releases` repo %%
-
+### Guides
+- [[Want some Sass in your obsidian theme‽ here's How and Why]]
```

Comment:
This shows up as a dangling link when I view the PR branch in Obsidian.

![image](https://user-images.githubusercontent.com/4840096/144301508-c4c58628-635c-4ce8-8135-1a11233d9b3f.png)

Reply:
Ah - I think in needs in -> with

![image](https://user-images.githubusercontent.com/4840096/144301643-bd2aadda-96ac-4f20-8b1d-ebf66537e71d.png)

Reply:
I'm very happy to fix the above after merging...

Reply:
Oh, I forgot about that! I can make a quick commit, no trouble. :)

Reply:
Both things fixed - thank you!